kiwi/bootloader/config/zipl.py

```diff
@@ -101,29 +100,9 @@ def write(self):
             with open(config_file, 'w') as config:
                 config.write(self.config)
 
-            log.info('Moving initrd/kernel to zipl boot directory')
-            Command.run(
-                [
-                    'mv',
-                    os.sep.join(
-                        [
-                            self.boot_dir, 'boot',
-                            os.readlink(self.boot_dir + '/boot/initrd')
-                        ]
-                    ),
-                    os.sep.join(
-                        [
-                            self.boot_dir, 'boot',
-                            os.readlink(self.boot_dir + '/boot/image')
-                        ]
-                    ),
-                    self._get_zipl_boot_path()
-                ]
-            )
-
     def setup_disk_image_config(
         self, boot_uuid=None, root_uuid=None, hypervisor=None,
-        kernel=None, initrd=None, boot_options=''
+        kernel='image', initrd='initrd', boot_options={}
```

Comment:
`boot_options` is not used in the whole method. I assume this is on purpose and the change here is done just to align with the definition of the same method in `bootloader/config/grub2.py`. I am fine with it, but I don't see the need as the common background for these two classes is `bootloader/config/base.py` where this method does not define any default value. Thus the default value is specific to the specialized class. If unused I'd probably set here `None` change the param definition to `:param dict boot_options: unused`. Anyway I can live with the dict default value if prefer it.

Also I just realized that the sphinx API docs are wrong for this method, the definition of `boot_options` is wrong in all three: base class, zipl class and grub2 class. This should be fix.

Reply:
good catch, I updated the api docs to match the implementation. I also changed the baseclass definition to use the same default value than any other specialized implementations. I want this method to be initialized with an empty dict because implementation code will access it as such.

I hope now it makes more sense ?